src/UpdatedPackagesCollector.js

```diff
@@ -101,6 +103,10 @@ export default class UpdatedPackagesCollector {
         .forEach(registerUpdated);
     }
 
+    if (Object.keys(updatedPackages).length > 0 && !this.repository.isIndependent()) {
```

Comment:
This isn't sufficient to determine the correct case, which is actually only necessary for _major_ version bumps. The current test is also insufficient for this case.